src/exporters/s5.py

```diff
@@ -0,0 +1,492 @@
+        else:
+            self.dataset = dataset
+
+        # get the reference dictionary that corresponds to that dataset
+        self.dataset_reference = dataset_reference[self.dataset]
+
+        # get the product type if it exists
+        self.product_type = self.get_product_type(product_type)
+
+        # check the product_type is valid for this dataset
+        if self.dataset_reference["product_type"] is not None:
+            assert (
+                self.product_type in self.dataset_reference["product_type"]
+            ), f"\
+                {self.product_type} is not a valid variable for the {self.dataset} dataset.\
+                Try one of: {self.dataset_reference['product_type']}"
+
+    def export(
+        self,
+        variable: str,
+        min_year: int = 2017,
+        max_year: int = 2018,
+        min_month: int = 1,
+        max_month: int = 12,
+        max_leadtime: Optional[int] = None,
+        pressure_levels: Optional[List[int]] = None,
+        selection_request: Optional[Dict] = None,
+        N_parallel_requests: int = 3,
+        show_api_request: bool = True,
+    ):
+        """
+        Arguments
+        --------
+        min_year: Optional[int] default = 2017
+            the minimum year of your request
+
+        max_year: Optional[int] default = 2018
+            the maximum year of your request
+
+        min_month: Optional[int] default = 1
+            the minimum month of your request
+
+        max_month: Optional[int] default = 12
+            the maximum month of your request
+
+        max_leadtime: Optional[int]
+            the maximum leadtime of your request
+                (if granularity is `hourly` then provide in days)
+                (elif granularity is `monthly` then provide in months)
+            defaults to ~3 months (90 days)
+
+        pressure_levels: Optional[int]
+            Pressure levels to download data at
+
+        Note:
+        ----
+        - All parameters that are assigned to class attributes are fixed for one download
+        - these are required to initialise the object [granularity, pressure_level]
+        - Only time will be chunked (by months) to send separate calls to the cdsapi
+        """
+        # N_parallel_requests can only be a MINIMUM of 1
+        if N_parallel_requests < 1:
+            N_parallel_requests = 1
+
+        # max_leadtime defaults
+        if max_leadtime is None:
+            # set the max_leadtime to 3 months as default
+            max_leadtime = 90 if (self.granularity == "hourly") else 3
+
+        if pressure_levels is None:
+            # set the pressure_levels to ['200', '500', '925'] as default
+            pressure_levels = [200, 500, 925] if (self.pressure_level) else None
+
+        processed_selection_request = self.create_selection_request(
+            variable=variable,
+            max_leadtime=max_leadtime,
+            min_year=min_year,
```

Comment:
The values for `min_year` differ between different functions. For the export function, it is 2018, for `get_s5_initialization_times` they are 1993, for `create_selection_request` it is 1993. Can this be normalized?

Reply:
done!